components/match2/commons/match_group_input.lua

```diff
@@ -583,6 +649,146 @@ function MatchGroupInput.readMvp(match)
 	return {players = parsedPlayers, points = mvppoints}
 end
 
+---reads the players of a team from input and wiki variables
+---@param match table
+---@param opponentIndex integer
+---@param opponent standardOpponent
+---@param options MatchGroupInputReadPlayersOfTeamOptions?
+---@param substitutionsInput string?
+---@return standardPlayer[]
+---@return table
+function MatchGroupInput.readPlayersOfTeamNew(match, opponentIndex, opponent, options, substitutionsInput)
```

Comment:
note to my future self: check if we have to check for opponent.name with and without underscores